sync/trypush.py

```diff
@@ -389,29 +392,67 @@ def wpt_tasks(self, force_update=False):
         self._data["tasks"] = wpt_tasks
         return wpt_tasks
 
+    def retrigger_failures(self, count=_retrigger_count):
+        wpt_tasks = self.wpt_tasks()
+        failures = [task.get("status", {}).get("state", None) != "success" for task in wpt_tasks]
```

Comment:
I feat that will try to retrigger things that are unscheduled. Isn't "failed" the only state we want to retrigger?

Reply:
I guess failed or exception. You're right that this might pick up stuff like unscheduled and pending.